### PR TITLE
Add the ability to configure the `cyhy-commander`'s `next-scan-limit` value

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This pull request adds the ability to configure the `next-scan-limit` value in the [cyhy-commander] configuration file.

## 💭 Motivation and context ##

We want to update the value being deployed in production but rather than continuously modifying a hard-coded value it made sense to integrate this as a configurable value between the Ansible role and the Terraform configuration.

## 🧪 Testing ##

Automated tests pass locally. Tests on GitHub will require final resolution of #661 and integration in this branch to pass. I deployed this in my development environment with a non-default value and confirmed it deployed successfully.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [ ] All new and existing tests pass.

[cyhy-commander]: https://github.com/cisagov/cyhy-commander

### CODE REVIEW

File: ansible/roles/cyhy_commander/defaults/main.yml [hunk 1]
```diff
@@ -1,2 +1,10 @@
 ---
 # defaults file for cyhy_commander
+
```
Comment (dav3r) [inline ansible/roles/cyhy_commander/defaults/main.yml:3]: Generally, we just delete this comment, though I'm fine if you'd rather handle that change wholesale for all of the ansible roles in here.
```suggestion
```

See [here](https://github.com/search?q=org%3Acisagov+defaults%2Fmain&type=code) for examples.

**Review by dav3r — APPROVED** [review]
This all works for me! 🖖 

I noted a nitpick, but feel free to ignore.

File: ansible/roles/cyhy_commander/defaults/main.yml (hunk 1 above)
Comment (mcdonnnj) [reply ansible/roles/cyhy_commander/defaults/main.yml:3]: If it's no issue for you I'd rather just update all of the roles wholesale as I'm sure there are similar comments in more than just the `defaults/main.yml` files.

**Review by jsf9k — APPROVED** [review]

**Comment by mcdonnnj** [comment]
@jsf9k @dav3r I modified this after looking at it again to change the `production` section instead of the `DEFAULT` section. I think this better aligns with what is intended and will also align with future updates to allow further customization through the Terraform configuration.

**Review by jsf9k — APPROVED** [review]

**Review by jasonodoom — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
👍 👍

File: ansible/roles/cyhy_commander/defaults/main.yml (hunk 1 above)
Comment (dav3r) [reply ansible/roles/cyhy_commander/defaults/main.yml:3]: That's fine by me.

**Pull request merged**